qaframework-bdd-tests/src/test/resources/features/refapp-2.x/02-login/verifyLogin.feature

```diff
@@ -0,0 +1,12 @@
+Feature: User based login
+
+@selenium
+@defaultLogin
+Scenario Outline: User Based login verification
+   When user verifies modules available on home page
+   And user verifies modules available on homepage after login as clerk
+   And user verifies modules available on home page after login as doctor
+   And user verifies modules available on home page after login as nurse
+   And user verifies modules available on home page after login as sysadmin
+   Then system goes back to login page
+   
```

Comment:
are you able to remove the white spaces from the EOF line?

Reply:
@kdaud  could you explain more of your statement above to make it more clear here.